THE-app/app/(tabs)/booksScreen.tsx

```diff
@@ -1,5 +1,6 @@
-import React from 'react';
-import { ScrollView, StyleSheet } from 'react-native';
+import React, { useState } from 'react';
+import { ScrollView, StyleSheet, TextInput } from 'react-native';
+import { Select } from "native-base";
```

Comment:
`native-base` is an abandoned library, and left a bad taste in the mouth of RN developers, because the author ignored reports of very bad performance issues for months, then decided to start a new library from scratch, leaving devs with few options but to rewrite a big part of their app.